### PR TITLE
Increase logging during integration tests

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Hopefully will help us spot the flaky failure #158 

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
When I read this  PR title I thought on 
```bash
  --ginkgo.v           
    If set, emits more output including GinkgoWriter contents.
  --ginkgo.vv   
    If set, emits with maximal verbosity - includes skipped and pending tests.
```

File: pkg/scheduler/scheduler.go [hunk 1]
```diff
@@ -397,11 +397,10 @@ func (e entryOrdering) Less(i, j int) bool {
...
 func (s *Scheduler) requeueAndUpdate(log logr.Logger, ctx context.Context, w *workload.Info, message string) {
-	if s.queues.RequeueWorkload(ctx, w) {
-		log.V(2).Info("Workload re-queued", "queuedWorkload", klog.KObj(w.Obj), "queue", klog.KRef(w.Obj.Namespace, w.Obj.Spec.QueueName))
-	}
+	existed := s.queues.RequeueWorkload(ctx, w)
+	log.V(2).Info("Workload re-queued", "queuedWorkload", klog.KObj(w.Obj), "queue", klog.KRef(w.Obj.Namespace, w.Obj.Spec.QueueName), "existed", existed)
 	err := workload.UpdateWorkloadStatus(ctx, s.client, w.Obj, kueue.QueuedWorkloadAdmitted, corev1.ConditionFalse, "Pending", message)
 	if err != nil {
-		log.Error(err, "Updating QueuedWorkload status")
+		log.Error(err, "Failed updating QueuedWorkload status")
```
Comment (ArangoGutierrez) [inline pkg/scheduler/scheduler.go:404]: as @ahg-g recommended, the `err` already carries the bad news, the side message is about "when" it failed

Comment (alculquicondor) [reply pkg/scheduler/scheduler.go]: Modified it in a better way, following https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message

**Review by alculquicondor — COMMENTED** [review]
I tried with `ginko.vv`, but it didn't affect zap

**Comment by alculquicondor** [comment]
/retest

Although it already helped me identify the bug :)

**Pull request merged**